samtranslator/model/api/api_generator.py

```diff
@@ -555,6 +555,24 @@ def _openapi_postprocess(self, definition_body):
                 components['schemas'] = definition_body['definitions']
                 definition_body['components'] = components
                 del definition_body['definitions']
+
+            if definition_body.get("paths"):
```

Comment:
The check for openapi3.x.x is already present in `_openapi_postprocess` so I haven't added that.
Updated the code with removing `consumes` and `produces`. Also, updated the code with `schema` for headers as per ApiGateway export output